erpnext/accounts/doctype/payment_entry/tests/test_payment_against_purchase_invoice.js

```diff
@@ -0,0 +1,42 @@
+QUnit.module('Payment Entry');
+
+QUnit.test("test payment entry", function(assert) {
+	assert.expect(5);
+	let done = assert.async();
+
+	frappe.run_serially([
+		() => {
+			return frappe.tests.make('Payment Entry', [
+				{payment_type:'Pay'},
+				{company:'For Testing'},
+				{party_type:'Supplier'},
+				{party:'Test Supplier'},
+				{paid_from:'Cash - FT'}
+			]);
+		},
+		() => cur_frm.save(),
+		() => frappe.tests.click_button('Submit'),
+		() => frappe.tests.click_button('Yes'),
+		() => frappe.timeout(2),
+		() => frappe.timeout(3),
+		() => {
+			assert.equal(cur_frm.doc.party, 'Test Supplier',
+				'supplier set in payment entry');
+			assert.equal(cur_frm.doc.paid_amount, 590,
```

Comment:
paid_amount can be anything, it depends on outstanding purchase invoices. If we add/edit some test cases which changes outstanding amount  of purchase invoices, it will fail.

Reply:
Try to rewrite this like https://github.com/frappe/erpnext/blob/develop/erpnext/accounts/doctype/payment_entry/tests/test_payment_against_invoice.js

Reply:
@nabinhait Okay.